pkg/sharding/ranges_test.go

```diff
@@ -94,3 +97,62 @@ func TestLogRanges_ResolveVirtualIndex(t *testing.T) {
 		}
 	}
 }
+
+func TestPublicKey(t *testing.T) {
+	ranges := LogRanges{
+		active: 45,
+		inactive: []LogRange{
+			{
+				TreeID:           10,
+				TreeLength:       10,
+				decodedPublicKey: "sharding",
+			}, {
+				TreeID:     20,
+				TreeLength: 20,
+			},
+		},
+	}
+	activePubKey := "activekey"
+	tests := []struct {
+		description    string
+		treeID         string
+		expectedPubKey string
+		shouldErr      bool
+	}{
+		{
+			description:    "empty tree ID",
+			expectedPubKey: "activekey",
+		}, {
+			description:    "tree id with encoded public key",
```

Comment:
```suggestion
			description:    "tree id with decoded public key",
```
Since we're skipping the encoding/decoding and that tree has decoded pubkey only it seems better to describe that way.